app/components/ui/icons/button-icons.tsx

```diff
@@ -0,0 +1,15 @@
+// Leading icons
+
+export function KeyIcon() {
```

Comment:
Should we move this into the existing icon folder?

Reply:
Let's do it! We have the icons file.